sympy/simplify/cse_main.py

```diff
@@ -555,6 +744,9 @@ def cse(exprs, symbols=None, optimizations=None, postprocess=None,
             reduced_exprs[i] = m
 
     if postprocess is None:
-        return replacements, reduced_exprs
+        result = replacements, reduced_exprs
+        print(result[1].args[1].args)
```

Comment:
Adding this print statement here, makes the test failing in the previous commit pass. Any ideas

Reply:
It doesn't. Ignore my above comment